.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,44 @@
-## Description
+**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be compeleted as written. Provide known gaps to testing that may raise the risk of merging to production.
 
 
-## Original issue(s)
-department-of-veterans-affairs/va.gov-team#0000
```

Comment:
Keep the inclusion of this link department-of-veterans-affairs/va.gov-team#0000. All tickets created in Zenhub will be on the va.gov-team repository